docs/resources/network_policy.md

```diff
@@ -12,12 +12,21 @@ description: |-
 ## Example Usage
 
 ```terraform
+resource "snowflake_network_rule" "rule" {
+  name       = "rule"
+  database   = "EXAMPLE_DB"
+  schema     = "EXAMPLE_SCHEMA"
+  comment    = "A rule."
+  type       = "IPV4"
+  mode       = "INGRESS"
+  value_list = ["192.168.0.100/24", "29.254.123.20"]
+}
+
 resource "snowflake_network_policy" "policy" {
   name    = "policy"
   comment = "A policy."
 
-  allowed_ip_list = ["192.168.0.100/24"]
-  blocked_ip_list = ["192.168.0.101"]
+  allowed_network_rule_list = [snowflake_network_rule.rule.qualified_name]
```

Comment:
It would be nice to have both examples in place (the previous one with allowed/disallowed IPs and the new one with the network rule list).

Reply:
Sure, done.